runtime/queries/php_only/indents.scm

```diff
@@ -12,6 +13,12 @@
   (case_statement)
 ] @indent.begin
 
+(return_statement [
+  (object_creation_expression)
+  (anonymous_function_creation_expression)
+  (match_expression)
+]) @indent.dedent
```

Comment:
You mean dedent for the inner expression, or the outer?

Reply:
The outer (return statement)---the inner expression is properly indented